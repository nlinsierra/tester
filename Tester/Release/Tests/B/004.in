QC 4S 2H KD 3D 7C JH 3H JS KS 2S 5D 6D TD 7H JD AS 9H 7S 4D 4H 8S 6C 4C JC 5S AC AH 7D TH TC
