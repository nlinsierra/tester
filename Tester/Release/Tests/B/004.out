21
2C 3C 5C 8C 9C KC 2D 8D 9D QD AD 5H 6H 8H QH KH 3S 6S 9S TS QS
